warehouse/manage/forms.py

```diff
@@ -87,12 +87,13 @@ def __init__(self, *args, user_service, **kwargs):
         self.user_service = user_service
 
 
-class DeleteTOTPForm(UsernameMixin, forms.Form):
+class DeleteTOTPForm(PasswordMixin, forms.Form):
 
-    __params__ = ["confirm_username"]
+    __params__ = ["confirm_password"]
 
-    def __init__(self, *args, user_service, **kwargs):
+    def __init__(self, *args, username, user_service, **kwargs):
         super().__init__(*args, **kwargs)
+        self.username = username
```

Comment:
Why not just make this a regular form field?

Reply:
I didn't think of that to be honest, probably because the actual HTML form only has a password, but it does make sense to have both mixins and manually add the username in the views. Thanks 👍